src/main/resources/files/readme.md.tpl

```diff
@@ -112,7 +112,21 @@ The example calls one endpoint from another JAX-RS resource where generated Rest
 [# th:if="${mp_graphql}"]
 ### GraphQL
 
-GraphQL is a query language for APIs and a library for fulfilling those queries with your existing data. Specification [here](https://microprofile.io/project/eclipse/microprofile-graphql)
+GraphQL is a remote data query language initially invented by Facebook and now evolving under it's own specification and community. MicroProfile GraphQL provides annotation-based APIs for building GraphQL services in Java. The specification is available [here](https://microprofile.io/project/eclipse/microprofile-graphql).
```

Comment:
Talking with @rdebusscher ... Can you replace 

> GraphQL is a remote data query language initially invented by Facebook and now evolving under it's own specification and community.

with 
GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data.

Reply:
Sounds good.  Done.